aries-site/src/examples/templates/dashboards/DashboardExample.js

```diff
@@ -1,6 +1,212 @@
+    subTitle: 'Events, Integration, and Status',
+    message: '204,353',
+  },
+  {
+    color: 'orange',
+    icon: <Location />,
+    title: 'Beacons',
+    subTitle: 'Uniqe identification light',
+    message: '24 beacons connected',
+  },
+  {
+    color: 'teal',
+    icon: <ShieldSecurity />,
+    title: 'Security',
+    subTitle: 'Trusted Platform Module',
+    message: 'No Module Connected',
+  },
+];
+
+const DashboardTiles = () => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Tiles
+      gap="medium"
+      columns={size !== 'small' ? 'small' : 'auto'}
+      margin={size === 'small' ? { bottom: 'xlarge' } : undefined}
+    >
+      {data.map(value => (
+        <Tile
+          background={value.color}
+          key={`Tile ${value.title}`}
+          alignContent="center"
+          onClick={() => {}}
+        >
+          <Identifier
+            pad={{ horizontal: 'medium', vertical: 'small' }}
+            title={value.title}
+            subTitle={value.subTitle}
+            size="small"
+            gap="medium"
+            direction="column"
+            align="start"
+          >
+            {value.icon}
+          </Identifier>
+          <Box flex />
+          <Footer pad="small" background="background-contrast">
+            <Text size="xsmall">{value.message}</Text>
+            {value.message === 'Connected' && (
+              <Box round pad="xsmall" background="green" />
+            )}
+          </Footer>
+        </Tile>
+      ))}
+    </Tiles>
+  );
+};
+
+const SidebarExample = ({ ...rest }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box
+      background="blue"
+      direction={size !== 'small' ? 'column' : 'row'}
+      pad={{
+        horizontal: size !== 'small' ? 'small' : 'medium',
+        vertical: size !== 'small' ? 'medium' : 'small',
+      }}
+      fill={size !== 'small' ? 'vertical' : 'horizontal'}
+      {...rest}
+    >
+      {size !== 'small' && (
+        <Avatar
+          margin={{ bottom: 'medium' }}
+          src="//s.gravatar.com/avatar/b7fb138d53ba0f573212ccce38a7c43b?s=80"
+        />
+      )}
+      <Box
+        flex="grow"
+        direction={size !== 'small' ? 'column' : 'row'}
+        justify={size === 'small' ? 'between' : undefined}
+      >
+        <Button a11yTitle="Clock" icon={<Clock color="text-xweak" />} />
+        <Button a11yTitle="Apps" icon={<Apps color="text-xweak" />} />
+        <Button a11yTitle="Terminal" icon={<Terminal color="text-xweak" />} />
+      </Box>
+
+      {size !== 'small' && (
+        <Box direction={size !== 'small' ? 'column' : 'row'}>
+          <Button a11yTitle="Chat" icon={<Chat color="text-xweak" />} />
+          <Button
+            a11yTitle="Help"
+            icon={<StatusUnknown color="text-xweak" />}
+          />
+        </Box>
+      )}
+    </Box>
+  );
+};
+
+const ScreenContainer = ({ mobile, ...rest }) => (
+  <ResponsiveContext.Provider value={mobile && 'small'}>
```

Comment:
This code is really only relevant to the way that the docs site toggles between desktop and mobile view for template examples. For someone who copy and pastes this out to their code, it would be irrelevant and probably cause confusion.

It feels like we will need to address this in the future.

Reply:
Or maybe, this ScreenContainer should live inside Example.js instead? and if a `template` prop is given to an Example, then we wrap the example in this? I would have to look more into implementation but could be an option.

Reply:
☝️